pootle/apps/pootle_store/views.py

```diff
@@ -362,11 +378,81 @@ def get_units(request):
         'store__translation_project__project',
         'store__translation_project__language',
     )
-    step_queryset = get_step_query(request, units_qs)
```

Comment:
Is there any reason not to remove `get_step_query` now that you copied all of its code here?